devutils/src/devutils/invoke_utils.py

```diff
@@ -390,30 +445,63 @@ def make_release(ctx):
             echo=True,
         )
 
-        print(f"Trigger the release with: git push origin {current_tag}")
+        print(f"Pushing tag: {current_tag}")
+        run(ctx, f"git push origin {current_tag}")
```

Comment:
Post-merge, you can test this out by making a release that we really need for `excel` (which I missed to do), which will produce this tag: `robocorp-excel-0.4.3`

As currently `inv check-tag-version` in `excel`:
```
Version does not match (robocorp-excel: 0.4.3 != repo tag: 0.4.2).
Tags:['robocorp-excel-0.3.0', 'robocorp-excel-0.4.0', 'robocorp-excel-0.4.1', 'robocorp-excel-0.4.2']
(exit(1))
```

So this is a good opportunity to test the command after the Merge.